apps/sms/index.html

```diff
@@ -84,6 +85,21 @@
     <script defer src="services/js/threads.js"></script>
     <script defer src="views/inbox/js/inbox.js"></script>
     <script defer src="views/shared/js/settings.js"></script>
+    <script defer src="lib/bridge/bridge.js"></script>
+    <script defer src="lib/bridge/plugins/stream/client.js"></script>
+    <script defer src="lib/bridge/plugins/stream/service.js"></script>
+    <script defer src="lib/bridge/plugins/finalize/client.js"></script>
+    <script defer src="services/js/bridge_service_mixin.js"></script>
+    <script defer src="services/js/moz_mobile_connections/moz_mobile_connections_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_shim.js"></script>
+    <script defer src="services/js/moz_settings/moz_settings_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_client.js"></script>
+    <script defer src="services/js/conversation/conversation_service.js"></script>
+    <script defer src="services/js/conversation/conversation_client.js"></script>
+    <script defer src="services/js/messaging/messaging_service.js"></script>
```

Comment:
I've moved MessagingService to the main file list to just check how bad it's in terms of performance to have MessaginService-in-window right away. So measuring. If it's bad we can move it to lazy loading back.